README.md

```diff
@@ -67,6 +67,7 @@ See [spec/README.md](spec/README.md). Summary:
     *   [Provenance]: To describe the origins of a software artifact.
     *   [Link]: For migration from [in-toto 0.9].
     *   [SPDX]: A Software Package Data Exchange document.
+*   [Bundles]: Defines a method of grouping multiple attestations together.
```

Comment:
nit: use singular for consistency with the other entries in the list. Same in other README and for the filename (bundle.md).

Reply:
Done.